Storage documentation

g4s8 opened the pull request:
For #860 - described FS and S3 storages, added todo for others, described default storages

olenagerasimova commented on .wiki/Configuration-Storage.md:
```diff
@@ -1 +1,70 @@
+Artipie "Storage" is an abstraction on top of multiple key-value storage providers. Artipie supports:
+ - file system storage
+ - S3 storage
+ - etcd storage (see limitations)
```
@g4s8 seems like word `limitations` requires some link

olenagerasimova reviewed (changes requested):
@g4s8 thanks, just one comment

g4s8 commented:
@olenagerasimova thanks, described missed storages and limitations

olenagerasimova commented:
@g4s8 please, could you check that push passed? Do not see any changes since previous push

g4s8 commented:
> @g4s8 please, could you check that push passed? Do not see any changes since previous push

@olenagerasimova sorry, didn't push, please see latest https://github.com/artipie/artipie/pull/916/commits/fdae7d8e56abcf405dd266bb69058f3b8a9b586e

olenagerasimova reviewed (approved):
@g4s8 thanks, approving
